socorro/processor/signature_utilities.py

```diff
@@ -683,6 +685,10 @@ def _create_frame_list(
         return frame_signatures_list
 
     #--------------------------------------------------------------------------
+    def _get_crashing_thread(self, processed_crash):
+        return processed_crash.json_dump['crash_info']['crashing_thread']
```

Comment:
E.g. https://github.com/mozilla/socorro/commit/8b321c1fbf068e897e484aa1c5a020854f39fa56#diff-a7d1c53ffed84fe4a5fee1abc9bd61eaR212
It's very possible I don't know what I'm talking about.